pandas/core/frame.py

```diff
@@ -3569,6 +3570,11 @@ def _get_value(self, index, col, takeable: bool = False) -> Scalar:
         Returns
         -------
         scalar
+
+        Notes
+        -----
+        Assumes that index and columns both have ax._index_as_unique;
```

Comment:
don't we have this as a base Index property? (or it can raise NotImplementedError), e.g. dont' like the AttributeError checking here.

Reply:
ill rephrase, it isn't that they have the attribute, it is that this is only called if `self.index._index_as_unique and self.columns._index_as_unique`